src/EventStore.ClusterNode/ClusterVNodeHostedService.cs

```diff
@@ -134,10 +104,19 @@ internal class ClusterVNodeHostedService : EventStoreHostedService<ClusterNodeOp
 					"\n========================================================================================================\n" +
 					"DEVELOPMENT MODE IS ON. THIS MODE IS *NOT* INTENDED FOR PRODUCTION USE.\n" +
 					"WHEN IN DEVELOPMENT MODE EVENTSTOREDB WILL\n" +
-					" - DISABLE TLS ON ALL TCP/HTTP INTERFACES.\n" +
+					" - DISABLE TLS ON ALL TCP INTERFACES.\n" +
 					" - ENABLE THE ATOMPUB PROTOCOL OVER HTTP.\n" +
 					"========================================================================================================\n");
 			}
+			else if (opts.Insecure) {
+				Log.Warning(
+					"\n========================================================================================================\n" +
+					"INSECURE MODE IS ON. THIS MODE IS *NOT* INTENDED FOR PRODUCTION USE.\n" +
```

Comment:
Is `Insecure` not a valid production use case?

A couple of lines below in the log we log out `It is recommended to run with TLS enabled in production.` as part of the `SECURITY` section.

We could instead log out
`"INSECURE MODE IS ON. THIS MODE IS *NOT* RECOMMENDED FOR PRODUCTION USE`.

Reply:
@pgermishuys thanks, I've made the change :+1: